src/theme/styles/index.css

```diff
@@ -0,0 +1,4 @@
+@import './base.css';
+@import './navbar.css';
+@import './sliderCarousel.css';
+@import './beforeAfter.css';
```

Comment:
Add end line

Reply:
Hmmmmm, there is an end line in my file, but it isn`t shown here